optimum/onnxruntime/base.py

```diff
@@ -41,9 +42,11 @@ def __init__(
     ):
         self.session = session
         self.parent_model = parent_model
-        self.normalized_config = NormalizedConfigManager.get_normalized_config_class(
-            self.parent_model.config.model_type
-        )(self.parent_model.config)
+        self.normalized_config = None
+        if isinstance(self.parent_model.config, PretrainedConfig):
+            self.normalized_config = NormalizedConfigManager.get_normalized_config_class(
+                self.parent_model.config.model_type
+            )(self.parent_model.config)
```

Comment:
I do not like `self.normalized_config` possibly being `None` because we would not expect that. I get that this is for Stable Diffusion. Could we make a subclass specifically for StableDiffusion?